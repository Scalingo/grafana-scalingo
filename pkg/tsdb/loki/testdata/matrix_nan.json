{
  "status": "success",
  "data": {
    "resultType": "matrix",
    "result": [
      {
        "metric": {},
        "values": [
          [1643014450.417, "NaN"],
          [1643014750.417, "NaN"]
        ]
      }
    ]
  }
}
