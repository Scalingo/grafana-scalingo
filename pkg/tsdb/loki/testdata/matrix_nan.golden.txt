ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {
    "executedQueryString": "Expr: up(ALERTS)\nStep: 42s"
}
Name: {}
Dimensions: 2 Fields by 2 Rows
+-----------------------------------+-----------------+
| Name: time                        | Name: value     |
| Labels:                           | Labels:         |
| Type: []time.Time                 | Type: []float64 |
+-----------------------------------+-----------------+
| 2022-01-24 08:54:10.417 +0000 UTC | NaN             |
| 2022-01-24 08:59:10.417 +0000 UTC | NaN             |
+-----------------------------------+-----------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////UAIAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEEAAoADAAAAAgABAAKAAAACAAAAKgAAAADAAAATAAAACgAAAAEAAAATP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAABs/v//CAAAAAwAAAACAAAAe30AAAQAAABuYW1lAAAAAIz+//8IAAAAQAAAADUAAAB7ImV4ZWN1dGVkUXVlcnlTdHJpbmciOiJFeHByOiB1cChBTEVSVFMpXG5TdGVwOiA0MnMifQAAAAQAAABtZXRhAAAAAAIAAADUAAAABAAAAEb///8UAAAAnAAAAJwAAAAAAAADnAAAAAMAAABQAAAALAAAAAQAAAAU////CAAAABAAAAAFAAAAdmFsdWUAAAAEAAAAbmFtZQAAAAA4////CAAAAAwAAAACAAAAe30AAAYAAABsYWJlbHMAAFj///8IAAAAJAAAABoAAAB7ImRpc3BsYXlOYW1lRnJvbURTIjoie30ifQAABgAAAGNvbmZpZwAAAAAAAFb///8AAAIABQAAAHZhbHVlABIAGAAUAAAAEwAMAAAACAAEABIAAAAUAAAAeAAAAIAAAAAAAAAKgAAAAAIAAAA0AAAABAAAANz///8IAAAAEAAAAAQAAAB0aW1lAAAAAAQAAABuYW1lAAAAAAgADAAIAAQACAAAAAgAAAAcAAAAEgAAAHsiaW50ZXJ2YWwiOjQyMDAwfQAABgAAAGNvbmZpZwAAAAAAAAAABgAIAAYABgAAAAAAAwAEAAAAdGltZQAAAAD/////uAAAABQAAAAAAAAADAAWABQAEwAMAAQADAAAACAAAAAAAAAAFAAAAAAAAAMEAAoAGAAMAAgABAAKAAAAFAAAAFgAAAACAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAAAAAAAACAAAAAgAAAAAAAAAAAAAAAAAAAAIAAAAAAAAAAAAAAAAAAABAXvAF8CjNFkAWVd81Kc0WAQAAAAAA+H8BAAAAAAD4fxAAAAAMABQAEgAMAAgABAAMAAAAEAAAACwAAAA8AAAAAAAEAAEAAABgAgAAAAAAAMAAAAAAAAAAIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAoADAAAAAgABAAKAAAACAAAAKgAAAADAAAATAAAACgAAAAEAAAATP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAABs/v//CAAAAAwAAAACAAAAe30AAAQAAABuYW1lAAAAAIz+//8IAAAAQAAAADUAAAB7ImV4ZWN1dGVkUXVlcnlTdHJpbmciOiJFeHByOiB1cChBTEVSVFMpXG5TdGVwOiA0MnMifQAAAAQAAABtZXRhAAAAAAIAAADUAAAABAAAAEb///8UAAAAnAAAAJwAAAAAAAADnAAAAAMAAABQAAAALAAAAAQAAAAU////CAAAABAAAAAFAAAAdmFsdWUAAAAEAAAAbmFtZQAAAAA4////CAAAAAwAAAACAAAAe30AAAYAAABsYWJlbHMAAFj///8IAAAAJAAAABoAAAB7ImRpc3BsYXlOYW1lRnJvbURTIjoie30ifQAABgAAAGNvbmZpZwAAAAAAAFb///8AAAIABQAAAHZhbHVlABIAGAAUAAAAEwAMAAAACAAEABIAAAAUAAAAeAAAAIAAAAAAAAAKgAAAAAIAAAA0AAAABAAAANz///8IAAAAEAAAAAQAAAB0aW1lAAAAAAQAAABuYW1lAAAAAAgADAAIAAQACAAAAAgAAAAcAAAAEgAAAHsiaW50ZXJ2YWwiOjQyMDAwfQAABgAAAGNvbmZpZwAAAAAAAAAABgAIAAYABgAAAAAAAwAEAAAAdGltZQAAAACAAgAAQVJST1cx
